client/app/lib/value-format.js

```diff
@@ -35,15 +35,27 @@ function createBooleanFormatter(values) {
   if (_.isArray(values)) {
     if (values.length >= 2) {
       // Both `true` and `false` specified
-      return value => '' + values[value ? 1 : 0];
+      return (value) => {
+        if (value === null) {
+          return 'null';
```

Comment:
Shouldn't we use empty string instead of `'null'`, like we do for numeric columns? And, BTW, here should be check for `undefined` too (the same as below)